<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSelectorPage</class>
 <widget class="QWizardPage" name="DeviceSelectorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Select a Device</string>
  </property>
  <property name="subTitle">
   <string>Please select the type of device you will use with QUsb2Snes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QRadioButton" name="sd2snesButton">
     <property name="styleSheet">
      <string notr="true">Text-align:left;
</string>
     </property>
     <property name="text">
      <string>A Physical console (SNES, SFC, SuperNT)
 with a Sd2Snes or FxPak pro.</string>
     </property>
     <property name="icon">
      <iconset resource="../../ressources.qrc">
       <normaloff>:/wizardpic/ikaripic.png</normaloff>:/wizardpic/ikaripic.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="retroarchButton">
     <property name="styleSheet">
      <string notr="true">Text-align:left</string>
     </property>
     <property name="text">
      <string>The RetroArch Emulator</string>
     </property>
     <property name="icon">
      <iconset resource="../../ressources.qrc">
       <normaloff>:/img/retroarch.png</normaloff>:/img/retroarch.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="nwaButton">
     <property name="styleSheet">
      <string notr="true">Text-align:left</string>
     </property>
     <property name="text">
      <string>Emulator with Emulator Network Access support</string>
     </property>
     <property name="icon">
      <iconset resource="../../ressources.qrc">
       <normaloff>:/img/mr chip.png</normaloff>:/img/mr chip.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="snesClassicButton">
     <property name="styleSheet">
      <string notr="true">Text-align:left</string>
     </property>
     <property name="text">
      <string>SNES Classic mini</string>
     </property>
     <property name="icon">
      <iconset resource="../../ressources.qrc">
       <normaloff>:/wizardpic/snesclassic.png</normaloff>:/wizardpic/snesclassic.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
